### DISCUSSION TITLE
Extra pcb

### DISCUSSION THREAD

**Discussion by jscottneal** [nebhead/PiFire#150] [General]
I ordered 5 boards and only need 2 of them. Have both button board and 4 input version.  Happy to share for cost of shipping.

**Comment by amackellar** [comment]
If still available, I would be interested in 2 boards!

**Comment by jscottneal** [comment]
Sure thing.  Give me your address and happy to send your way.

**Reply by amackellar** [reply]
Would I be able to email you? I dont want to post my address on the disscussion board.

**Comment by jscottneal** [comment]
For sure.  ***@***.***

**Reply by amackellar** [reply]
Hhmm... Github seems to sensor email addresses. I wish there was just a direct message option but it does not exist. 

Can you attach a screenshot of your email?

**Comment by jscottneal** [comment]
go to our website and submit info in form. It will come to me
https://cdtvineyard.com/contact/

**Reply by nebhead** [reply]
> go to our website and submit info in form. It will come to me https://cdtvineyard.com/contact/

Now you have my attention. :)  OK, the next time I'm up visiting family in Oregon, I'll have to swing through and do a tasting there!    

BTW - On this thread, I wanted to mention that our Discord(https://discord.gg/F9mbCrbrZS) has a hardware sharing channel for board sharing.  So if you are looking for different versions of the PCB or button boards, etc, that is a great place to check with others.

**Reply by amackellar** [reply]
Ive tried submitting info a few times on different browsers but get a "Refresh page and try again" message.

Heres an image of my email address, if you could send me a line I'll reply with my mailing address.

![image](https://user-images.githubusercontent.com/60200911/229883165-f64f3d85-ba58-4748-bdd8-e384d79c6f31.png)

**Comment by jscottneal** [comment]
Give it another shot. Sorry for delay

Scott Neal
Owner/Winemaker
503-883-4111 ext. 210
*www.cdtvineyard.com <http://www.cdtvineyard.com/>*

**Comment by carkom** [comment]
You still have extras?

**Comment by jscottneal** [comment]
I do
